components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIManagerConfiguration.java

```diff
@@ -179,7 +198,39 @@ private void readChildElements(OMElement serverConfig,
             OMElement element = (OMElement) childElements.next();
             String localName = element.getLocalName();
             nameStack.push(localName);
-            if (elementHasText(element)) {
+            if ("TokenRevocationNotifiers".equals(localName)) {
+                tokenRevocationClassName = element.getAttributeValue(new QName("class"));
+                tokenRevocationNotifiers = new LinkedHashMap<>();
+            } else if ("RealtimeNotifier".equals(localName)) {
+                Iterator revocationPropertiesIterator = element.getChildrenWithLocalName("Property");
+                Properties properties = new Properties();
+                while (revocationPropertiesIterator.hasNext()) {
+                    OMElement propertyElem = (OMElement) revocationPropertiesIterator.next();
+
+                    properties.setProperty(propertyElem.getAttributeValue(new QName("name")), propertyElem.getText());
+                }
+                tokenRevocationNotifiers.put(localName, properties);
+            } else if ("PersistentNotifier".equals(localName)) {
+                Iterator revocationPropertiesIterator = element.getChildrenWithLocalName("Property");
+                Properties properties = new Properties();
+                while (revocationPropertiesIterator.hasNext()) {
+                    OMElement propertyElem = (OMElement) revocationPropertiesIterator.next();
+                    if (propertyElem.getAttributeValue(new QName("name")).equalsIgnoreCase("password")) {
```

Comment:
can't we do this without hard coding `password`? as @tharindu1st mentioned?

Reply:
yeah, i was waiting for the secure-vault update in carbon-apimgt before changing the code. I will change this accordingly.